FE-Smart_Watering_System/src/services/SensorServices.js

```
@@ -131,22 +196,24 @@ class SensorServices {
                 
               case 'pump_water':
                 endpoint = API_ENDPOINTS.DEVICES.GET_PUMP_WATER(device.id);
-                const pumpSpeed = savedData && savedData.pumpWater ? savedData.pumpWater.speed  : 0;
+                const pumpSpeed = savedData && savedData.pumpWater ? savedData.pumpWater.speed : 0;
                 fallbackData = {
                   deviceType: 'pump_water',
                   deviceId: device.id,
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)